### DISCUSSION TITLE
Расширить ItemActions

### DISCUSSION THREAD

**Discussion by k-rustemuly** [moonshine-software/moonshine#462] [Ideas]
![delete](https://github.com/moonshine-software/moonshine/assets/16911003/2be25125-2f4a-4d99-9b89-d7bcaae2f91f)
Добавить для confirm такую возможность как показано на рисунке.

**Comment by lee-to** [comment]
Можно будет легко реализовать в 2.0
В 1 уже не будем добавлять
